.github/workflows/go.yml

```diff
@@ -0,0 +1,53 @@
+name: Go
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+        
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
+      
+    - name: Install dependencies
+      run: |
+        go get google.golang.org/grpc/cmd/protoc-gen-go-grpc
+        go get github.com/golang/protobuf/protoc-gen-go   
+        go get golang.org/x/lint/golint
```

Comment:
`make deps` already includes part of this, maybe makes sense including what is missing and call it here?

You could also consider using https://github.com/actions/cache for caching the go packages.

Reply:
ok

Reply:
I added make deps call, but I do not see how the cache would actually help ...

Reply:
Caching the go packages would make the next runs faster as they would need to download only the outdated packages, if any.